docs/source/operator_reference.rst

```diff
@@ -4,23 +4,48 @@
 Operators
 =========
 
-Loading operators.
-
-.. raw:: html
-
-    <iframe
-      src="_static/dpf.html#CPython"
-      style="
-        position: fixed;
-        top: 36px;
-        bottom: 0px;
-        right: 0px;
-        width: 100%;
-        border: none;
-        margin: 0;
-        padding: 0;
-        overflow: hidden;
-        z-index: 1000;
-        height: 100%;
-      ">
-    </iframe>
+DPF operators allow the manipulation and the transformation of simulation data.
```

Comment:
```suggestion
DPF operators are used to manipulate and transform simulation data.
```

Reply:
@PProfizi You can use provide in the way written--or change it to:
DPF operators are used to manipulate and transform simulation data.